ReactNativeClient/lib/models/Folder.js

```diff
@@ -107,15 +107,18 @@ class Folder extends BaseItem {
 
 	// Calculates note counts for all folders and adds the note_count attribute to each folder
 	// Note: this only calculates the overall number of nodes for this folder and all its descendants
-	static async addNoteCounts(folders) {
+	static async addNoteCounts(folders, showDone) {
```

Comment:
For consistency, please name the new parameter `includeCompletedTodos` with default value `true`